packages/chakra-ui/src/components/ui/number-input.tsx

```diff
@@ -0,0 +1,21 @@
+import { NumberInput as ChakraNumberInput } from '@chakra-ui/react';
+import * as React from 'react';
```

Comment:
Two things. First, put the react imports up top. Secondly, don't use the `import * as React from 'react';` pattern. Instead import only that which you need. Like:
`import { forwardRef } from 'react'`;